std_msgs/Header header
float64 volume_cmd            # volume_cmd reading
float64 volume_cmd_variance   # 0 is interpreted as unknown

